packages/fast-components-styles-msft/src/action-trigger/index.ts

```diff
@@ -0,0 +1,153 @@
+    );
+    const secondaryDisabledBackgroundColor: string = disabledContrast(
+        contrastScale,
+        secondaryBackgroundColor,
+        backgroundColor
+    );
+    const secondaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        foregroundColor,
+        designSystem.brandColor
+    );
+    // Define primary button colors
+    const primaryRestBackgroundColor: string = scaledEnsureNormalContrast(
+        scaledEnsureNormalContrast(brandColor, backgroundColor),
+        color
+    );
+    const primaryDisabledBackground: string = disabledContrast(
+        contrastScale,
+        primaryRestBackgroundColor,
+        backgroundColor
+    );
+    const primaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        color,
+        primaryDisabledBackground
+    );
+    const outlineColor: string = scaledEnsureNormalContrast(
+        foregroundColor,
+        backgroundColor
+    );
+    const outlineBorderColor: string = scaledNormalContrast(
+        foregroundColor,
+        backgroundColor
+    );
+    const outlineDisabledColor: string = disabledContrast(
+        designSystem.contrast,
+        outlineColor,
+        backgroundColor
+    );
+    const outlineDisabledBorderColor: string = outlineDisabledColor;
+    return {
+        actionTrigger: {
+            display: "inline-flex",
+            maxWidth: "100%",
+            minWidth: "14px",
+        },
+        actionTrigger_glyph: {
+            display: "inline-block",
+            position: "relative",
+            width: "8px",
+            [applyLocalizedProperty("marginRight", "marginLeft", direction)]: "6px",
+        },
+        actionTrigger__primary: {
+            "& $actionTrigger_glyph": {
+                fill: color,
+            },
+            "&$actionTrigger__disabled $actionTrigger_glyph": {
+                fill: primaryDisabledColor,
+            },
+        },
+        actionTrigger__lightweight: {
+            "& $actionTrigger_glyph": {
+                fill: primaryRestBackgroundColor,
+            },
+            "&$actionTrigger__disabled $actionTrigger_glyph": {
+                fill: secondaryDisabledColor,
+            },
+        },
+        actionTrigger__justified: {
+            [applyLocalizedProperty("marginLeft", "marginRight", direction)]: "-10px",
+            "& $actionTrigger_glyph": {
+                fill: primaryRestBackgroundColor,
+            },
+            "&$actionTrigger__disabled $actionTrigger_glyph": {
+                fill: secondaryDisabledColor,
+            },
+        },
+        actionTrigger__outline: {
+            [applyLocalizedProperty("marginLeft", "marginRight", direction)]: "-10px",
+            "& $actionTrigger_glyph": {
+                fill: outlineColor,
+            },
+            "&$actionTrigger__disabled $actionTrigger_glyph": {
+                fill: outlineDisabledColor,
+            },
+        },
+        actionTrigger__disabled: {},
```

Comment:
Should the `fill: outlineDisabledColor,` actually be here?

Reply:
Yes, otherwise an svg glyph won't pick up the disabled color.